src/main/java/Nave/TaskList.java

```diff
@@ -0,0 +1,68 @@
+package Nave;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> list;
```

Comment:
Typically we try to avoid using the variable name ```list``` as it could cause confusion with ```java.util.List```